INFO 2024-06-03 20:14:01,117 autoreload 9668 2520 Watching for file changes with StatReloader
INFO 2024-06-03 20:14:08,721 views 9668 1328 Index page accessed
INFO 2024-06-03 20:14:08,722 basehttp 9668 1328 "GET / HTTP/1.1" 200 40
WARNING 2024-06-03 20:14:12,454 log 9668 1328 Not Found: /docs
WARNING 2024-06-03 20:14:12,455 basehttp 9668 1328 "GET /docs HTTP/1.1" 404 2960
WARNING 2024-06-03 20:14:15,857 log 9668 1328 Not Found: /coin/
WARNING 2024-06-03 20:14:15,858 basehttp 9668 1328 "GET /coin/ HTTP/1.1" 404 2963
INFO 2024-06-03 20:14:43,061 basehttp 9668 1328 "GET /random_coin HTTP/1.1" 301 0
INFO 2024-06-03 20:14:43,064 basehttp 9668 380 "GET /random_coin/ HTTP/1.1" 200 74
INFO 2024-06-03 20:14:47,113 basehttp 9668 380 "GET /random_coin/ HTTP/1.1" 200 76
INFO 2024-06-03 20:16:30,020 autoreload 9668 2520 D:\Ucheba\django_GB\myproject\myapp\views.py changed, reloading.
INFO 2024-06-03 20:16:30,535 autoreload 11620 5920 Watching for file changes with StatReloader
INFO 2024-06-03 20:16:34,664 basehttp 11620 12368 "GET /random_number HTTP/1.1" 301 0
DEBUG 2024-06-03 20:16:34,666 views 11620 3504 About page accessed
INFO 2024-06-03 20:16:34,667 basehttp 11620 3504 "GET /random_number/ HTTP/1.1" 200 57
INFO 2024-06-03 20:18:18,196 autoreload 11620 5920 D:\Ucheba\django_GB\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-03 20:19:07,937 autoreload 15244 9188 Watching for file changes with StatReloader
INFO 2024-06-03 20:19:12,899 basehttp 15244 10132 "GET / HTTP/1.1" 200 40
WARNING 2024-06-03 20:19:15,578 log 15244 10132 Not Found: /docs
WARNING 2024-06-03 20:19:15,578 basehttp 15244 10132 "GET /docs HTTP/1.1" 404 2960
INFO 2024-06-03 20:19:17,208 basehttp 15244 10132 "GET /random_number/ HTTP/1.1" 200 57
INFO 2024-06-03 20:19:21,607 basehttp 15244 10132 "GET /random_number/ HTTP/1.1" 200 57
WARNING 2024-06-03 20:19:26,359 log 15244 10132 Not Found: /random_number_22/
WARNING 2024-06-03 20:19:26,360 basehttp 15244 10132 "GET /random_number_22/ HTTP/1.1" 404 2999
INFO 2024-06-03 20:20:39,800 autoreload 15244 9188 D:\Ucheba\django_GB\myproject\myapp\views.py changed, reloading.
INFO 2024-06-03 20:20:40,302 autoreload 5028 12768 Watching for file changes with StatReloader
INFO 2024-06-03 21:24:21,847 autoreload 5028 12768 D:\Ucheba\django_GB\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-03 21:24:22,430 autoreload 15524 1332 Watching for file changes with StatReloader
INFO 2024-06-03 21:31:02,227 autoreload 15856 6492 Watching for file changes with StatReloader
ERROR 2024-06-03 21:31:04,047 exception 15856 13888 Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
WARNING 2024-06-03 21:31:04,098 log 15856 13888 Bad Request: /
WARNING 2024-06-03 21:31:04,098 basehttp 15856 13888 "GET / HTTP/1.1" 400 72332
ERROR 2024-06-03 21:31:06,606 exception 15856 1612 Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
WARNING 2024-06-03 21:31:06,620 log 15856 1612 Bad Request: /
WARNING 2024-06-03 21:31:06,621 basehttp 15856 1612 "GET / HTTP/1.1" 400 72469
INFO 2024-06-03 21:31:39,741 autoreload 9816 10808 Watching for file changes with StatReloader
ERROR 2024-06-03 21:31:40,763 log 9816 14172 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'main/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: main/index.html
ERROR 2024-06-03 21:31:40,765 basehttp 9816 14172 "GET / HTTP/1.1" 500 81639
INFO 2024-06-03 21:33:05,483 autoreload 9816 10808 D:\Ucheba\django_GB\myproject\main\views.py changed, reloading.
INFO 2024-06-03 21:33:05,952 autoreload 15508 7420 Watching for file changes with StatReloader
ERROR 2024-06-03 21:33:11,834 log 15508 5188 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, './main/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: ./main/index.html
ERROR 2024-06-03 21:33:11,836 basehttp 15508 5188 "GET / HTTP/1.1" 500 81802
INFO 2024-06-03 21:33:37,707 autoreload 15508 7420 D:\Ucheba\django_GB\myproject\main\views.py changed, reloading.
INFO 2024-06-03 21:33:38,205 autoreload 6456 6728 Watching for file changes with StatReloader
ERROR 2024-06-03 21:33:39,339 log 6456 3200 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2024-06-03 21:33:39,341 basehttp 6456 3200 "GET / HTTP/1.1" 500 81681
ERROR 2024-06-03 21:33:40,224 log 6456 3200 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2024-06-03 21:33:40,225 basehttp 6456 3200 "GET / HTTP/1.1" 500 81681
ERROR 2024-06-03 21:33:40,753 log 6456 3200 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2024-06-03 21:33:40,754 basehttp 6456 3200 "GET / HTTP/1.1" 500 81681
INFO 2024-06-03 21:34:23,310 autoreload 6456 6728 D:\Ucheba\django_GB\myproject\main\views.py changed, reloading.
INFO 2024-06-03 21:34:23,793 autoreload 15704 13432 Watching for file changes with StatReloader
ERROR 2024-06-03 21:38:42,858 log 15704 16288 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'main/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: main/index.html
ERROR 2024-06-03 21:38:42,861 basehttp 15704 16288 "GET / HTTP/1.1" 500 81639
INFO 2024-06-03 21:38:43,269 autoreload 15704 13432 D:\Ucheba\django_GB\myproject\myproject\urls.py changed, reloading.
INFO 2024-06-03 21:38:43,846 autoreload 10664 14796 Watching for file changes with StatReloader
INFO 2024-06-03 21:39:00,433 autoreload 12960 4492 Watching for file changes with StatReloader
INFO 2024-06-03 21:39:02,327 basehttp 12960 15676 "GET / HTTP/1.1" 200 132
WARNING 2024-06-03 21:39:17,187 log 12960 15676 Not Found: /docs
WARNING 2024-06-03 21:39:17,188 basehttp 12960 15676 "GET /docs HTTP/1.1" 404 2546
INFO 2024-06-03 21:39:19,288 basehttp 12960 15676 "GET /about HTTP/1.1" 301 0
INFO 2024-06-03 21:39:19,291 basehttp 12960 5112 "GET /about/ HTTP/1.1" 200 131
INFO 2024-06-03 21:52:29,087 autoreload 6320 11352 Watching for file changes with StatReloader
INFO 2024-06-03 21:52:30,691 basehttp 6320 6336 "GET / HTTP/1.1" 200 132
INFO 2024-06-03 21:52:37,416 basehttp 6320 6336 "GET /about/ HTTP/1.1" 200 2783
INFO 2024-06-03 21:59:20,798 basehttp 6320 11860 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 21:59:20,818 basehttp 6320 11860 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 21:59:48,630 basehttp 6320 16016 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 21:59:48,644 basehttp 6320 16016 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 21:59:49,182 basehttp 6320 15940 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 21:59:49,195 basehttp 6320 15940 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 21:59:49,375 basehttp 6320 6700 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 21:59:49,388 basehttp 6320 6700 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 21:59:49,540 basehttp 6320 15340 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 21:59:49,553 basehttp 6320 15340 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 21:59:49,685 basehttp 6320 5724 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 21:59:49,700 basehttp 6320 5724 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 21:59:49,857 basehttp 6320 6276 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 21:59:49,871 basehttp 6320 6276 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:01:11,601 basehttp 6320 12164 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 22:01:11,616 basehttp 6320 12164 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:01:13,070 basehttp 6320 12724 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 22:01:13,085 basehttp 6320 12724 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:01:25,741 basehttp 6320 11320 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 22:01:25,754 basehttp 6320 11320 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:01:26,479 basehttp 6320 16076 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 22:01:26,516 basehttp 6320 16076 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:01:26,743 basehttp 6320 3720 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 22:01:26,758 basehttp 6320 3720 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:01:26,990 basehttp 6320 8316 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 22:01:27,005 basehttp 6320 8316 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:01:27,149 basehttp 6320 13524 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 22:01:27,163 basehttp 6320 13524 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:01:27,364 basehttp 6320 4624 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 22:01:27,376 basehttp 6320 4624 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:01:27,981 basehttp 6320 5016 "GET /about/ HTTP/1.1" 200 2765
WARNING 2024-06-03 22:01:28,012 basehttp 6320 5016 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:02:14,330 basehttp 6320 12520 "GET /about/ HTTP/1.1" 200 2778
WARNING 2024-06-03 22:02:14,346 basehttp 6320 12520 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:02:14,719 basehttp 6320 1488 "GET /about/ HTTP/1.1" 200 2778
WARNING 2024-06-03 22:02:14,732 basehttp 6320 1488 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:02:52,968 basehttp 6320 7984 "GET /about/ HTTP/1.1" 200 2767
WARNING 2024-06-03 22:02:52,985 basehttp 6320 7984 "GET /static/.main/img/my_foto.jpg HTTP/1.1" 404 1923
INFO 2024-06-03 22:02:53,334 basehttp 6320 14168 "GET /about/ HTTP/1.1" 200 2767
WARNING 2024-06-03 22:02:53,347 basehttp 6320 14168 "GET /static/.main/img/my_foto.jpg HTTP/1.1" 404 1923
INFO 2024-06-03 22:02:53,573 basehttp 6320 212 "GET /about/ HTTP/1.1" 200 2767
WARNING 2024-06-03 22:02:53,592 basehttp 6320 212 "GET /static/.main/img/my_foto.jpg HTTP/1.1" 404 1923
INFO 2024-06-03 22:02:53,718 basehttp 6320 15436 "GET /about/ HTTP/1.1" 200 2767
WARNING 2024-06-03 22:02:53,731 basehttp 6320 15436 "GET /static/.main/img/my_foto.jpg HTTP/1.1" 404 1923
INFO 2024-06-03 22:04:00,912 basehttp 6320 3200 "GET /about/ HTTP/1.1" 200 2766
WARNING 2024-06-03 22:04:00,927 basehttp 6320 3200 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:04:01,462 basehttp 6320 16372 "GET /about/ HTTP/1.1" 200 2766
WARNING 2024-06-03 22:04:01,475 basehttp 6320 16372 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:04:17,172 basehttp 6320 14868 "GET /about/ HTTP/1.1" 200 2766
WARNING 2024-06-03 22:04:17,190 basehttp 6320 14868 "GET /static/main/img/my_foto.jpg HTTP/1.1" 404 1920
INFO 2024-06-03 22:04:27,350 autoreload 8288 9112 Watching for file changes with StatReloader
INFO 2024-06-03 22:04:30,325 basehttp 8288 15404 "GET /about/ HTTP/1.1" 200 2766
INFO 2024-06-03 22:04:30,369 basehttp 8288 15404 "GET /static/main/img/my_foto.jpg HTTP/1.1" 200 23878
ERROR 2024-06-03 22:14:42,538 log 8288 264 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'main/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-03 22:14:42,542 basehttp 8288 264 "GET / HTTP/1.1" 500 136808
ERROR 2024-06-03 22:14:42,944 log 8288 264 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'main/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-03 22:14:42,946 basehttp 8288 264 "GET / HTTP/1.1" 500 136534
ERROR 2024-06-03 22:15:25,231 log 8288 264 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'main/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-03 22:15:25,233 basehttp 8288 264 "GET / HTTP/1.1" 500 136756
ERROR 2024-06-03 22:15:32,516 log 8288 264 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'main/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-03 22:15:32,519 basehttp 8288 264 "GET / HTTP/1.1" 500 136756
ERROR 2024-06-03 22:15:33,250 log 8288 264 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'main/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-03 22:15:33,253 basehttp 8288 264 "GET / HTTP/1.1" 500 136756
INFO 2024-06-03 22:16:08,749 autoreload 1096 13792 Watching for file changes with StatReloader
ERROR 2024-06-03 22:16:09,931 log 1096 10740 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\myproject\main\views.py", line 5, in index
    return render(request, 'main/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 481, in parse
    raise self.error(token, e)
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 479, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_GB\venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '('about')' from 'url_for('about')'
ERROR 2024-06-03 22:16:09,935 basehttp 1096 10740 "GET / HTTP/1.1" 500 139129
INFO 2024-06-03 22:20:28,485 basehttp 1096 10740 "GET / HTTP/1.1" 200 600
INFO 2024-06-03 22:20:33,717 basehttp 1096 10740 "GET /about/ HTTP/1.1" 200 2766
INFO 2024-06-03 22:20:39,940 basehttp 1096 10740 "GET /about/ HTTP/1.1" 200 2766
INFO 2024-06-03 22:21:25,143 basehttp 1096 10740 "GET / HTTP/1.1" 200 665
INFO 2024-06-03 22:21:25,838 basehttp 1096 10740 "GET / HTTP/1.1" 200 665
INFO 2024-06-03 22:21:26,309 basehttp 1096 10740 "GET / HTTP/1.1" 200 665
INFO 2024-06-03 22:21:26,799 basehttp 1096 10740 "GET / HTTP/1.1" 200 665
INFO 2024-06-03 22:21:34,124 autoreload 5152 15408 Watching for file changes with StatReloader
INFO 2024-06-03 22:21:35,586 basehttp 5152 15520 "GET / HTTP/1.1" 200 665
INFO 2024-06-03 22:21:36,941 basehttp 5152 15520 "GET /about/ HTTP/1.1" 200 2766
INFO 2024-06-03 22:21:39,707 basehttp 5152 15520 "GET /about/ HTTP/1.1" 200 2766
INFO 2024-06-03 22:21:58,507 basehttp 5152 15520 "GET /about/ HTTP/1.1" 200 2766
INFO 2024-06-03 22:22:16,593 basehttp 5152 15520 "GET / HTTP/1.1" 200 673
INFO 2024-06-03 22:23:40,055 basehttp 5152 15520 "GET / HTTP/1.1" 200 649
INFO 2024-06-03 22:23:40,564 basehttp 5152 15520 "GET / HTTP/1.1" 200 649
INFO 2024-06-03 22:23:40,740 basehttp 5152 15520 "GET / HTTP/1.1" 200 649
INFO 2024-06-03 22:23:40,909 basehttp 5152 15520 "GET / HTTP/1.1" 200 649
INFO 2024-06-03 22:26:26,477 basehttp 5152 15520 "GET / HTTP/1.1" 200 648
INFO 2024-06-03 22:26:32,033 basehttp 5152 15520 "GET /about/ HTTP/1.1" 200 2766
INFO 2024-06-03 22:26:34,924 basehttp 5152 15520 "GET /about/ HTTP/1.1" 200 2766
INFO 2024-06-03 22:27:12,343 basehttp 5152 15520 "GET /about/ HTTP/1.1" 200 2766
INFO 2024-06-03 22:28:48,323 autoreload 14592 16356 Watching for file changes with StatReloader
INFO 2024-06-03 22:28:49,712 basehttp 14592 6928 "GET / HTTP/1.1" 200 648
INFO 2024-06-03 22:28:51,100 basehttp 14592 6928 "GET /about/ HTTP/1.1" 200 2958
INFO 2024-06-03 22:28:52,441 basehttp 14592 6928 "GET / HTTP/1.1" 200 648
INFO 2024-06-03 22:28:53,744 basehttp 14592 6928 "GET /about/ HTTP/1.1" 200 2958
INFO 2024-06-03 22:29:16,800 autoreload 1572 8452 Watching for file changes with StatReloader
INFO 2024-06-03 22:29:22,227 basehttp 1572 8588 "GET / HTTP/1.1" 200 648
INFO 2024-06-03 22:29:23,139 basehttp 1572 8588 "GET /about/ HTTP/1.1" 200 2951
INFO 2024-06-03 22:29:24,018 basehttp 1572 8588 "GET / HTTP/1.1" 200 648
INFO 2024-06-03 22:29:25,377 basehttp 1572 8588 "GET /about/ HTTP/1.1" 200 2951
INFO 2024-06-03 22:29:26,231 basehttp 1572 8588 "GET / HTTP/1.1" 200 648
INFO 2024-06-03 22:29:27,594 basehttp 1572 8588 "GET /about/ HTTP/1.1" 200 2951
INFO 2024-06-03 22:29:54,914 basehttp 1572 8588 "GET / HTTP/1.1" 200 648
INFO 2024-06-08 09:24:27,242 autoreload 12504 16948 Watching for file changes with StatReloader
INFO 2024-06-08 09:25:07,896 basehttp 12504 7036 "GET / HTTP/1.1" 200 651
WARNING 2024-06-08 09:25:08,357 log 12504 7036 Not Found: /favicon.ico
WARNING 2024-06-08 09:25:08,358 basehttp 12504 7036 "GET /favicon.ico HTTP/1.1" 404 2773
WARNING 2024-06-08 09:25:55,745 log 12504 7400 Not Found: /wm/
WARNING 2024-06-08 09:25:55,745 basehttp 12504 7400 "GET /wm/ HTTP/1.1" 404 5538
WARNING 2024-06-08 09:26:18,550 log 12504 7400 Not Found: /wm/docs
WARNING 2024-06-08 09:26:18,550 basehttp 12504 7400 "GET /wm/docs HTTP/1.1" 404 5550
WARNING 2024-06-08 09:26:27,931 log 12504 7400 Not Found: /docs
WARNING 2024-06-08 09:26:27,931 basehttp 12504 7400 "GET /docs HTTP/1.1" 404 2752
WARNING 2024-06-08 09:26:28,398 log 12504 7400 Not Found: /docs
WARNING 2024-06-08 09:26:28,399 basehttp 12504 7400 "GET /docs HTTP/1.1" 404 2752
WARNING 2024-06-08 09:27:02,299 log 12504 7400 Not Found: /docs
WARNING 2024-06-08 09:27:02,300 basehttp 12504 7400 "GET /docs HTTP/1.1" 404 2752
INFO 2024-06-08 09:28:21,431 autoreload 12504 16948 D:\Ucheba\django_GB\myproject\myproject\urls.py changed, reloading.
INFO 2024-06-08 09:28:21,944 autoreload 14596 14156 Watching for file changes with StatReloader
WARNING 2024-06-08 09:28:22,075 log 14596 3460 Not Found: /docs
WARNING 2024-06-08 09:28:22,076 basehttp 14596 3460 "GET /docs HTTP/1.1" 404 2649
WARNING 2024-06-08 09:28:22,794 log 14596 3460 Not Found: /docs
WARNING 2024-06-08 09:28:22,794 basehttp 14596 3460 "GET /docs HTTP/1.1" 404 2649
WARNING 2024-06-08 09:28:44,417 log 14596 3460 Not Found: /docs
WARNING 2024-06-08 09:28:44,418 basehttp 14596 3460 "GET /docs HTTP/1.1" 404 2649
INFO 2024-06-08 09:28:45,311 autoreload 14596 14156 D:\Ucheba\django_GB\myproject\myproject\urls.py changed, reloading.
INFO 2024-06-08 09:28:45,829 autoreload 9840 9872 Watching for file changes with StatReloader
WARNING 2024-06-08 09:28:46,172 log 9840 10640 Not Found: /docs
WARNING 2024-06-08 09:28:46,173 basehttp 9840 10640 "GET /docs HTTP/1.1" 404 2546
WARNING 2024-06-08 09:28:48,408 log 9840 10640 Not Found: /docs
WARNING 2024-06-08 09:28:48,409 basehttp 9840 10640 "GET /docs HTTP/1.1" 404 2546
WARNING 2024-06-08 09:28:49,440 log 9840 10640 Not Found: /docs
WARNING 2024-06-08 09:28:49,440 basehttp 9840 10640 "GET /docs HTTP/1.1" 404 2546
WARNING 2024-06-08 09:28:54,103 log 9840 10640 Not Found: /docs
WARNING 2024-06-08 09:28:54,103 basehttp 9840 10640 "GET /docs HTTP/1.1" 404 2546
INFO 2024-06-08 09:28:54,493 autoreload 9840 9872 D:\Ucheba\django_GB\myproject\myproject\urls.py changed, reloading.
INFO 2024-06-08 09:28:55,005 autoreload 12776 12720 Watching for file changes with StatReloader
WARNING 2024-06-08 09:28:55,194 log 12776 17236 Not Found: /docs
WARNING 2024-06-08 09:28:55,195 basehttp 12776 17236 "GET /docs HTTP/1.1" 404 2219
WARNING 2024-06-08 09:28:57,586 log 12776 17236 Not Found: /docs
WARNING 2024-06-08 09:28:57,587 basehttp 12776 17236 "GET /docs HTTP/1.1" 404 2219
WARNING 2024-06-08 09:28:58,272 log 12776 17236 Not Found: /docs
WARNING 2024-06-08 09:28:58,273 basehttp 12776 17236 "GET /docs HTTP/1.1" 404 2219
INFO 2024-06-08 09:29:32,901 autoreload 12776 12720 D:\Ucheba\django_GB\myproject\myproject\urls.py changed, reloading.
INFO 2024-06-08 09:29:33,525 autoreload 4668 11976 Watching for file changes with StatReloader
WARNING 2024-06-08 09:29:33,710 log 4668 6456 Not Found: /docs
WARNING 2024-06-08 09:29:33,711 basehttp 4668 6456 "GET /docs HTTP/1.1" 404 2546
WARNING 2024-06-08 09:31:57,438 log 4668 6456 Not Found: /docs
WARNING 2024-06-08 09:31:57,439 basehttp 4668 6456 "GET /docs HTTP/1.1" 404 2546
WARNING 2024-06-08 09:31:58,065 log 4668 6456 Not Found: /docs
WARNING 2024-06-08 09:31:58,066 basehttp 4668 6456 "GET /docs HTTP/1.1" 404 2546
INFO 2024-06-08 09:32:05,956 basehttp 4668 6456 "GET / HTTP/1.1" 200 648
INFO 2024-06-08 09:32:19,735 basehttp 4668 6456 "GET /admin HTTP/1.1" 301 0
INFO 2024-06-08 09:32:19,753 basehttp 4668 1524 "GET /admin/ HTTP/1.1" 302 0
INFO 2024-06-08 09:32:19,897 basehttp 4668 1524 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4241
INFO 2024-06-08 09:32:20,039 basehttp 4668 11236 "GET /static/admin/css/responsive.css HTTP/1.1" 200 18529
INFO 2024-06-08 09:32:20,039 basehttp 4668 1524 "GET /static/admin/css/base.css HTTP/1.1" 200 21441
INFO 2024-06-08 09:32:20,045 basehttp 4668 13800 "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2682
INFO 2024-06-08 09:32:20,088 basehttp 4668 3236 "GET /static/admin/css/login.css HTTP/1.1" 200 958
INFO 2024-06-08 09:32:20,088 basehttp 4668 3244 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
INFO 2024-06-08 09:32:20,092 basehttp 4668 14928 "GET /static/admin/js/theme.js HTTP/1.1" 200 1943
INFO 2024-06-08 09:32:20,093 basehttp 4668 1524 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
INFO 2024-06-08 09:32:21,677 basehttp 4668 14928 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4508
WARNING 2024-06-08 09:32:21,782 log 4668 14928 Not Found: /favicon.ico
WARNING 2024-06-08 09:32:21,782 basehttp 4668 14928 "GET /favicon.ico HTTP/1.1" 404 2567
WARNING 2024-06-08 09:32:21,863 log 4668 14928 Not Found: /favicon.ico
WARNING 2024-06-08 09:32:21,864 basehttp 4668 14928 "GET /favicon.ico HTTP/1.1" 404 2567
INFO 2024-06-08 09:32:28,114 basehttp 4668 14928 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4508
WARNING 2024-06-08 09:32:28,151 log 4668 14928 Not Found: /favicon.ico
WARNING 2024-06-08 09:32:28,151 basehttp 4668 14928 "GET /favicon.ico HTTP/1.1" 404 2567
WARNING 2024-06-08 09:32:28,161 log 4668 14928 Not Found: /favicon.ico
WARNING 2024-06-08 09:32:28,161 basehttp 4668 14928 "GET /favicon.ico HTTP/1.1" 404 2567
INFO 2024-06-08 09:33:01,697 basehttp 4668 14928 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4508
WARNING 2024-06-08 09:33:01,742 log 4668 14928 Not Found: /favicon.ico
WARNING 2024-06-08 09:33:01,742 basehttp 4668 14928 "GET /favicon.ico HTTP/1.1" 404 2567
WARNING 2024-06-08 09:33:01,752 log 4668 14928 Not Found: /favicon.ico
WARNING 2024-06-08 09:33:01,752 basehttp 4668 14928 "GET /favicon.ico HTTP/1.1" 404 2567
INFO 2024-06-08 09:33:13,262 basehttp 4668 14928 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4508
WARNING 2024-06-08 09:33:13,299 log 4668 14928 Not Found: /favicon.ico
WARNING 2024-06-08 09:33:13,300 basehttp 4668 14928 "GET /favicon.ico HTTP/1.1" 404 2567
WARNING 2024-06-08 09:33:13,308 log 4668 14928 Not Found: /favicon.ico
WARNING 2024-06-08 09:33:13,309 basehttp 4668 14928 "GET /favicon.ico HTTP/1.1" 404 2567
INFO 2024-06-08 09:35:36,430 autoreload 4592 11552 Watching for file changes with StatReloader
INFO 2024-06-08 09:35:37,708 basehttp 4592 14924 "GET / HTTP/1.1" 200 648
WARNING 2024-06-08 09:35:40,322 log 4592 14924 Not Found: /docs
WARNING 2024-06-08 09:35:40,322 basehttp 4592 14924 "GET /docs HTTP/1.1" 404 2546
INFO 2024-06-08 09:35:44,385 basehttp 4592 14924 "GET /admin/ HTTP/1.1" 302 0
INFO 2024-06-08 09:35:44,410 basehttp 4592 14924 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4241
INFO 2024-06-08 09:35:55,001 basehttp 4592 14924 "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2024-06-08 09:35:55,044 basehttp 4592 14924 "GET /admin/ HTTP/1.1" 200 5826
INFO 2024-06-08 09:35:55,092 basehttp 4592 14924 "GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
INFO 2024-06-08 09:35:55,095 basehttp 4592 8948 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
INFO 2024-06-08 09:35:55,095 basehttp 4592 9352 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
WARNING 2024-06-08 09:35:55,119 log 4592 9352 Not Found: /favicon.ico
WARNING 2024-06-08 09:35:55,120 basehttp 4592 9352 "GET /favicon.ico HTTP/1.1" 404 2567
WARNING 2024-06-08 09:35:55,184 log 4592 9352 Not Found: /favicon.ico
WARNING 2024-06-08 09:35:55,184 basehttp 4592 9352 "GET /favicon.ico HTTP/1.1" 404 2567
INFO 2024-06-08 09:36:01,390 basehttp 4592 9352 "GET /admin/auth/group/ HTTP/1.1" 200 7629
INFO 2024-06-08 09:36:01,415 basehttp 4592 9352 "GET /static/admin/css/changelists.css HTTP/1.1" 200 6811
INFO 2024-06-08 09:36:01,418 basehttp 4592 8000 "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
INFO 2024-06-08 09:36:01,423 basehttp 4592 12264 "GET /static/admin/js/core.js HTTP/1.1" 200 6208
INFO 2024-06-08 09:36:01,430 basehttp 4592 8000 "GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
INFO 2024-06-08 09:36:01,430 basehttp 4592 9352 "GET /static/admin/js/actions.js HTTP/1.1" 200 8067
INFO 2024-06-08 09:36:01,437 basehttp 4592 8948 "GET /admin/jsi18n/ HTTP/1.1" 200 16065
INFO 2024-06-08 09:36:01,437 basehttp 4592 5144 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 9042
INFO 2024-06-08 09:36:01,439 basehttp 4592 12264 "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
INFO 2024-06-08 09:36:01,442 basehttp 4592 9352 "GET /static/admin/img/search.svg HTTP/1.1" 200 458
INFO 2024-06-08 09:36:01,467 basehttp 4592 14924 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 285314
INFO 2024-06-08 09:36:01,481 basehttp 4592 14924 "GET /static/admin/js/filters.js HTTP/1.1" 200 978
INFO 2024-06-08 09:36:01,496 basehttp 4592 8000 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 325171
INFO 2024-06-08 09:36:01,513 basehttp 4592 8000 "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
INFO 2024-06-08 09:36:02,694 basehttp 4592 8000 "GET /admin/auth/user/ HTTP/1.1" 200 11601
INFO 2024-06-08 09:36:02,708 basehttp 4592 8000 "GET /admin/jsi18n/ HTTP/1.1" 200 16065
INFO 2024-06-08 09:36:02,711 basehttp 4592 14924 "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
INFO 2024-06-08 09:36:02,738 basehttp 4592 8000 "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
INFO 2024-06-08 09:36:02,743 basehttp 4592 14924 "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
INFO 2024-06-08 09:36:04,504 basehttp 4592 14924 "GET /admin/auth/user/1/change/ HTTP/1.1" 200 23370
INFO 2024-06-08 09:36:04,520 basehttp 4592 5144 "GET /admin/jsi18n/ HTTP/1.1" 200 16065
INFO 2024-06-08 09:36:04,530 basehttp 4592 9352 "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 19319
INFO 2024-06-08 09:36:04,530 basehttp 4592 8000 "GET /static/admin/js/calendar.js HTTP/1.1" 200 9141
INFO 2024-06-08 09:36:04,531 basehttp 4592 14924 "GET /static/admin/css/forms.css HTTP/1.1" 200 9047
INFO 2024-06-08 09:36:04,531 basehttp 4592 12264 "GET /static/admin/js/SelectBox.js HTTP/1.1" 200 4530
INFO 2024-06-08 09:36:04,534 basehttp 4592 5144 "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
INFO 2024-06-08 09:36:04,535 basehttp 4592 8948 "GET /static/admin/js/SelectFilter2.js HTTP/1.1" 200 15292
INFO 2024-06-08 09:36:04,544 basehttp 4592 12264 "GET /static/admin/css/widgets.css HTTP/1.1" 200 11907
INFO 2024-06-08 09:36:04,549 basehttp 4592 8948 "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
INFO 2024-06-08 09:36:04,592 basehttp 4592 8948 "GET /static/admin/img/icon-unknown.svg HTTP/1.1" 200 655
INFO 2024-06-08 09:36:04,593 basehttp 4592 5144 "GET /static/admin/img/icon-unknown-alt.svg HTTP/1.1" 200 655
INFO 2024-06-08 09:36:04,594 basehttp 4592 8000 "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
INFO 2024-06-08 09:36:04,595 basehttp 4592 12264 "GET /static/admin/img/selector-icons.svg HTTP/1.1" 200 3291
INFO 2024-06-08 09:36:04,602 basehttp 4592 14924 "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
INFO 2024-06-08 09:36:06,662 basehttp 4592 14924 "GET /admin/auth/group/ HTTP/1.1" 200 7629
INFO 2024-06-08 09:36:06,675 basehttp 4592 14924 "GET /admin/jsi18n/ HTTP/1.1" 200 16065
INFO 2024-06-08 09:36:09,349 basehttp 4592 14924 "POST /admin/logout/ HTTP/1.1" 200 3683
INFO 2024-06-09 09:22:30,141 autoreload 8312 8392 Watching for file changes with StatReloader
INFO 2024-06-09 09:22:34,516 basehttp 8312 1312 "GET / HTTP/1.1" 200 648
INFO 2024-06-09 09:22:34,520 basehttp 8312 1312 "GET / HTTP/1.1" 200 648
WARNING 2024-06-09 09:22:34,794 log 8312 1312 Not Found: /favicon.ico
WARNING 2024-06-09 09:22:34,795 basehttp 8312 1312 "GET /favicon.ico HTTP/1.1" 404 2567
WARNING 2024-06-09 09:22:34,801 log 8312 11296 Not Found: /favicon.ico
WARNING 2024-06-09 09:22:34,802 basehttp 8312 11296 "GET /favicon.ico HTTP/1.1" 404 2567
WARNING 2024-06-09 09:23:03,035 log 8312 10576 Not Found: /clients/create/
WARNING 2024-06-09 09:23:03,035 basehttp 8312 10576 "GET /clients/create/ HTTP/1.1" 404 2579
WARNING 2024-06-09 09:23:45,150 log 8312 10576 Not Found: /clients/create/client-create
WARNING 2024-06-09 09:23:45,151 basehttp 8312 10576 "GET /clients/create/client-create HTTP/1.1" 404 2618
INFO 2024-06-10 22:15:53,876 autoreload 3512 1700 Watching for file changes with StatReloader
INFO 2024-06-10 22:15:56,967 basehttp 3512 344 "GET / HTTP/1.1" 200 648
WARNING 2024-06-10 22:15:57,124 log 3512 344 Not Found: /favicon.ico
WARNING 2024-06-10 22:15:57,124 basehttp 3512 344 "GET /favicon.ico HTTP/1.1" 404 2567
INFO 2024-06-10 22:15:58,682 basehttp 3512 344 "GET /about/ HTTP/1.1" 200 2951
INFO 2024-06-10 22:15:58,722 basehttp 3512 344 "GET /static/main/img/my_foto.jpg HTTP/1.1" 200 23878
INFO 2024-06-10 22:15:59,831 basehttp 3512 344 "GET / HTTP/1.1" 200 648
INFO 2024-06-10 22:19:13,373 autoreload 7296 14852 Watching for file changes with StatReloader
INFO 2024-06-11 22:03:07,852 autoreload 18072 11168 Watching for file changes with StatReloader
INFO 2024-06-11 22:03:15,335 basehttp 18072 13508 "GET / HTTP/1.1" 200 651
WARNING 2024-06-11 22:03:15,642 log 18072 13508 Not Found: /favicon.ico
WARNING 2024-06-11 22:03:15,643 basehttp 18072 13508 "GET /favicon.ico HTTP/1.1" 404 2774
WARNING 2024-06-11 22:03:46,397 log 18072 17820 Not Found: /lec/
WARNING 2024-06-11 22:03:46,397 basehttp 18072 17820 "GET /lec/ HTTP/1.1" 404 2993
INFO 2024-06-11 22:04:06,911 basehttp 18072 17820 "GET /lec/hello/ HTTP/1.1" 200 23
INFO 2024-06-11 22:04:14,818 basehttp 18072 17820 "GET /lec/hello2/ HTTP/1.1" 200 23
INFO 2024-06-11 22:09:20,196 autoreload 18072 11168 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-11 22:09:20,714 autoreload 10752 6896 Watching for file changes with StatReloader
INFO 2024-06-11 22:12:39,661 autoreload 10752 6896 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-11 22:12:40,545 autoreload 4824 7748 Watching for file changes with StatReloader
INFO 2024-06-11 22:16:59,993 autoreload 4824 7748 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-11 22:17:00,486 autoreload 12924 13128 Watching for file changes with StatReloader
INFO 2024-06-11 22:17:24,724 autoreload 12924 13128 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-11 22:17:25,253 autoreload 13788 10476 Watching for file changes with StatReloader
INFO 2024-06-11 22:17:46,923 autoreload 10948 8684 Watching for file changes with StatReloader
INFO 2024-06-11 22:18:44,309 autoreload 10948 8684 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-11 22:18:44,807 autoreload 7276 2372 Watching for file changes with StatReloader
WARNING 2024-06-11 22:18:46,995 log 7276 16644 Not Found: /lec/posts/
WARNING 2024-06-11 22:18:46,996 basehttp 7276 16644 "GET /lec/posts/ HTTP/1.1" 404 3222
WARNING 2024-06-11 22:19:14,762 log 7276 16644 Not Found: /lec/posts/2022
WARNING 2024-06-11 22:19:14,762 basehttp 7276 16644 "GET /lec/posts/2022 HTTP/1.1" 404 3234
INFO 2024-06-11 22:19:39,359 basehttp 7276 16644 "GET /lec/posts/2022/02 HTTP/1.1" 301 0
INFO 2024-06-11 22:19:39,361 basehttp 7276 1040 "GET /lec/posts/2022/02/ HTTP/1.1" 200 21
INFO 2024-06-11 22:22:04,866 autoreload 7276 2372 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-11 22:22:05,378 autoreload 17096 14560 Watching for file changes with StatReloader
INFO 2024-06-11 22:22:10,777 basehttp 17096 6896 "GET /lec/posts/2022/02/ HTTP/1.1" 200 21
INFO 2024-06-11 22:22:14,290 basehttp 17096 6896 "GET /lec/posts/2022 HTTP/1.1" 301 0
INFO 2024-06-11 22:22:14,292 basehttp 17096 3500 "GET /lec/posts/2022/ HTTP/1.1" 200 19
INFO 2024-06-11 22:23:17,018 basehttp 17096 3500 "GET /lec/posts/2022/6/ HTTP/1.1" 200 21
INFO 2024-06-11 22:28:07,741 autoreload 17096 14560 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-11 22:28:08,256 autoreload 18088 5660 Watching for file changes with StatReloader
INFO 2024-06-11 22:28:20,468 basehttp 18088 11076 "GET /lec/posts/2022/6/12/ HTTP/1.1" 200 352
INFO 2024-06-11 22:48:42,193 autoreload 18088 5660 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-11 22:48:42,714 autoreload 2492 10136 Watching for file changes with StatReloader
INFO 2024-06-11 22:50:12,599 autoreload 2492 10136 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-11 22:50:13,112 autoreload 17296 14608 Watching for file changes with StatReloader
INFO 2024-06-11 22:50:20,730 autoreload 17296 14608 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-11 22:50:21,246 autoreload 10440 11316 Watching for file changes with StatReloader
INFO 2024-06-11 22:51:11,427 autoreload 10440 11316 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-11 22:51:11,943 autoreload 16556 11656 Watching for file changes with StatReloader
INFO 2024-06-11 22:51:25,748 autoreload 16556 11656 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-11 22:51:26,265 autoreload 17064 12912 Watching for file changes with StatReloader
INFO 2024-06-11 22:51:30,183 basehttp 17064 12004 "GET /lec/ HTTP/1.1" 200 171
INFO 2024-06-11 23:02:39,952 autoreload 17064 12912 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-11 23:02:40,483 autoreload 14576 15408 Watching for file changes with StatReloader
INFO 2024-06-11 23:03:37,050 autoreload 14576 15408 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-11 23:03:37,549 autoreload 12804 7780 Watching for file changes with StatReloader
INFO 2024-06-12 16:34:36,807 autoreload 10904 10952 Watching for file changes with StatReloader
INFO 2024-06-12 16:34:39,535 basehttp 10904 3712 "GET / HTTP/1.1" 200 651
WARNING 2024-06-12 16:34:40,040 log 10904 3712 Not Found: /favicon.ico
WARNING 2024-06-12 16:34:40,043 basehttp 10904 3712 "GET /favicon.ico HTTP/1.1" 404 2774
WARNING 2024-06-12 16:34:40,057 log 10904 9148 Not Found: /favicon.ico
WARNING 2024-06-12 16:34:40,058 basehttp 10904 9148 "GET /favicon.ico HTTP/1.1" 404 2774
INFO 2024-06-12 16:34:51,270 basehttp 10904 13524 "GET /lec/ HTTP/1.1" 200 175
INFO 2024-06-12 16:38:32,132 autoreload 10904 10952 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-12 16:38:32,773 autoreload 13412 7824 Watching for file changes with StatReloader
INFO 2024-06-12 16:42:04,599 autoreload 13412 7824 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-12 16:42:05,093 autoreload 6428 14024 Watching for file changes with StatReloader
INFO 2024-06-12 16:42:49,340 autoreload 6428 14024 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-12 16:42:49,834 autoreload 13852 14164 Watching for file changes with StatReloader
INFO 2024-06-12 16:43:12,215 basehttp 13852 10668 "GET /lec/if/ HTTP/1.1" 200 334
INFO 2024-06-12 17:00:35,606 autoreload 13852 14164 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-12 17:00:36,114 autoreload 2036 11920 Watching for file changes with StatReloader
INFO 2024-06-12 17:02:37,769 autoreload 2036 11920 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-12 17:02:38,278 autoreload 1880 10336 Watching for file changes with StatReloader
INFO 2024-06-12 17:02:43,547 basehttp 1880 13376 "GET /lec/for/ HTTP/1.1" 200 963
INFO 2024-06-12 17:05:17,897 autoreload 1880 10336 D:\Ucheba\django_project\myproject\seminar_3\views.py changed, reloading.
INFO 2024-06-12 17:05:18,399 autoreload 13924 8760 Watching for file changes with StatReloader
INFO 2024-06-12 17:20:43,626 autoreload 13924 8760 D:\Ucheba\django_project\myproject\lection3\views.py changed, reloading.
INFO 2024-06-12 17:20:44,156 autoreload 12796 11516 Watching for file changes with StatReloader
INFO 2024-06-12 17:22:07,095 autoreload 12796 11516 D:\Ucheba\django_project\myproject\lection3\urls.py changed, reloading.
INFO 2024-06-12 17:22:07,589 autoreload 8780 3088 Watching for file changes with StatReloader
INFO 2024-06-12 17:22:33,762 basehttp 8780 13928 "GET /lec/ HTTP/1.1" 200 175
INFO 2024-06-12 17:22:38,477 basehttp 8780 13928 "GET /lec/index HTTP/1.1" 301 0
ERROR 2024-06-12 17:22:38,679 log 8780 12784 Internal Server Error: /lec/index/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'enblock'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\lection3\views.py", line 73, in index
    return render(request, "lection3/index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 2: 'enblock', expected 'endblock'. Did you forget to register or load this tag?
ERROR 2024-06-12 17:22:38,684 basehttp 8780 12784 "GET /lec/index/ HTTP/1.1" 500 163918
INFO 2024-06-12 17:23:48,699 basehttp 8780 12784 "GET /lec/index/ HTTP/1.1" 200 249
INFO 2024-06-12 17:23:55,938 basehttp 8780 12784 "GET /lec/about HTTP/1.1" 301 0
INFO 2024-06-12 17:23:55,940 basehttp 8780 6300 "GET /lec/about/ HTTP/1.1" 200 274
INFO 2024-06-12 17:25:59,447 autoreload 8780 3088 D:\Ucheba\django_project\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-12 17:25:59,953 autoreload 1684 13444 Watching for file changes with StatReloader
INFO 2024-06-12 17:26:19,054 autoreload 1684 13444 D:\Ucheba\django_project\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-12 17:26:19,560 autoreload 8108 12684 Watching for file changes with StatReloader
INFO 2024-06-12 17:29:23,087 autoreload 8108 12684 D:\Ucheba\django_project\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-12 17:29:23,652 autoreload 14244 13336 Watching for file changes with StatReloader
INFO 2024-06-12 17:30:33,547 basehttp 14244 1820 "GET /lec/about/ HTTP/1.1" 200 334
INFO 2024-06-12 17:30:48,174 basehttp 14244 1820 "GET /lec/index/ HTTP/1.1" 200 309
INFO 2024-06-12 17:33:22,722 autoreload 14244 13336 D:\Ucheba\django_project\myproject\myproject\urls.py changed, reloading.
INFO 2024-06-12 17:33:23,250 autoreload 13260 8268 Watching for file changes with StatReloader
INFO 2024-06-12 17:34:26,171 autoreload 13260 8268 D:\Ucheba\django_project\myproject\myproject\urls.py changed, reloading.
INFO 2024-06-12 17:34:26,657 autoreload 11576 3652 Watching for file changes with StatReloader
INFO 2024-06-12 17:34:43,554 autoreload 10680 1544 Watching for file changes with StatReloader
ERROR 2024-06-12 17:34:44,585 log 10680 8764 Internal Server Error: /
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\main\views.py", line 5, in index
    return render(request, 'main/my_template.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: main/my_template.html
ERROR 2024-06-12 17:34:44,586 basehttp 10680 8764 "GET / HTTP/1.1" 500 83604
INFO 2024-06-12 17:36:53,712 autoreload 10680 1544 D:\Ucheba\django_project\myproject\main\views.py changed, reloading.
INFO 2024-06-12 17:36:54,230 autoreload 14216 1224 Watching for file changes with StatReloader
INFO 2024-06-12 17:36:55,089 basehttp 14216 12976 "GET / HTTP/1.1" 200 652
INFO 2024-06-12 17:36:56,532 basehttp 14216 12976 "GET /lec/about/ HTTP/1.1" 200 274
INFO 2024-06-12 17:37:00,898 basehttp 14216 12976 "GET /lec/about/ HTTP/1.1" 200 274
INFO 2024-06-12 17:38:05,941 basehttp 14216 12976 "GET /lec/about/ HTTP/1.1" 200 274
INFO 2024-06-12 17:40:10,192 autoreload 14216 1224 D:\Ucheba\django_project\myproject\main\views.py changed, reloading.
INFO 2024-06-12 17:40:10,764 autoreload 3600 10180 Watching for file changes with StatReloader
INFO 2024-06-12 17:40:17,683 basehttp 3600 12540 "GET / HTTP/1.1" 200 652
INFO 2024-06-12 17:40:19,165 basehttp 3600 12540 "GET /lec/about/ HTTP/1.1" 200 274
INFO 2024-06-12 17:40:21,826 basehttp 3600 12540 "GET / HTTP/1.1" 200 652
INFO 2024-06-12 17:40:22,544 basehttp 3600 12540 "GET /lec/about/ HTTP/1.1" 200 274
INFO 2024-06-12 17:40:24,779 basehttp 3600 12540 "GET /lec/about/ HTTP/1.1" 200 274
INFO 2024-06-12 17:41:40,144 autoreload 3600 10180 D:\Ucheba\django_project\myproject\main\urls.py changed, reloading.
INFO 2024-06-12 17:41:40,647 autoreload 11172 12068 Watching for file changes with StatReloader
INFO 2024-06-12 17:41:41,072 basehttp 11172 10592 "GET / HTTP/1.1" 200 648
INFO 2024-06-12 17:41:42,167 basehttp 11172 10592 "GET /about/ HTTP/1.1" 200 2951
INFO 2024-06-12 17:41:42,242 basehttp 11172 10592 "GET /static/main/img/my_foto.jpg HTTP/1.1" 200 23878
INFO 2024-06-12 17:41:44,627 basehttp 11172 10592 "GET /about/ HTTP/1.1" 200 2951
INFO 2024-06-12 17:41:45,515 basehttp 11172 10592 "GET / HTTP/1.1" 200 648
INFO 2024-06-12 17:41:46,489 basehttp 11172 10592 "GET /about/ HTTP/1.1" 200 2951
INFO 2024-06-12 17:41:47,418 basehttp 11172 10592 "GET / HTTP/1.1" 200 648
INFO 2024-06-12 17:41:50,195 basehttp 11172 10592 "GET /about/ HTTP/1.1" 200 2951
INFO 2024-06-12 17:41:51,410 basehttp 11172 10592 "GET / HTTP/1.1" 200 648
INFO 2024-06-12 17:41:52,187 basehttp 11172 10592 "GET /about/ HTTP/1.1" 200 2951
INFO 2024-06-12 18:50:23,764 autoreload 14456 11120 Watching for file changes with StatReloader
INFO 2024-06-12 18:50:27,149 basehttp 14456 16356 "GET / HTTP/1.1" 200 648
INFO 2024-06-12 18:50:34,605 basehttp 14456 16356 "GET /lec/ HTTP/1.1" 200 175
WARNING 2024-06-12 18:50:45,949 log 14456 16356 Not Found: /lec/author/
WARNING 2024-06-12 18:50:45,951 basehttp 14456 16356 "GET /lec/author/ HTTP/1.1" 404 4777
ERROR 2024-06-12 18:51:10,071 log 14456 16356 Internal Server Error: /lec/author/11/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\lection3\views.py", line 85, in author_posts
    return render(request, "lection3/author_posts.html", {"author": author, "posts": posts})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: lection3/author_posts.html
ERROR 2024-06-12 18:51:10,073 basehttp 14456 16356 "GET /lec/author/11/ HTTP/1.1" 500 86088
INFO 2024-06-12 19:02:16,255 basehttp 14456 14880 "GET /lec/author/11/ HTTP/1.1" 200 858
INFO 2024-06-12 19:02:42,049 basehttp 14456 14880 "GET /lec/author/5 HTTP/1.1" 301 0
INFO 2024-06-12 19:02:42,052 basehttp 14456 7020 "GET /lec/author/5/ HTTP/1.1" 200 853
INFO 2024-06-12 19:02:47,669 basehttp 14456 7020 "GET /lec/post/50/ HTTP/1.1" 200 746
INFO 2024-06-12 19:02:52,387 basehttp 14456 7020 "GET /lec/post/48/ HTTP/1.1" 200 697
INFO 2024-06-12 19:03:04,049 basehttp 14456 7020 "GET /lec/posts/12 HTTP/1.1" 301 0
INFO 2024-06-12 19:03:04,051 basehttp 14456 7052 "GET /lec/posts/12/ HTTP/1.1" 200 17
INFO 2024-06-12 19:03:33,738 basehttp 14456 7052 "GET /lec/post/46/ HTTP/1.1" 200 723
INFO 2024-06-12 19:04:59,889 basehttp 14456 7052 "GET /lec/post/12 HTTP/1.1" 301 0
INFO 2024-06-12 19:04:59,892 basehttp 14456 8644 "GET /lec/post/12/ HTTP/1.1" 200 678
INFO 2024-06-12 19:05:05,822 basehttp 14456 8644 "GET /lec/post/1 HTTP/1.1" 301 0
INFO 2024-06-12 19:05:05,825 basehttp 14456 6120 "GET /lec/post/1/ HTTP/1.1" 200 761
INFO 2024-06-12 19:47:37,071 autoreload 1220 15872 Watching for file changes with StatReloader
INFO 2024-06-12 19:48:58,558 autoreload 12264 2892 Watching for file changes with StatReloader
INFO 2024-06-12 19:49:31,344 autoreload 12264 2892 D:\Ucheba\django_project\myproject\myproject\urls.py changed, reloading.
INFO 2024-06-12 19:49:31,875 autoreload 12796 6120 Watching for file changes with StatReloader
INFO 2024-06-12 21:27:27,642 autoreload 13988 15708 Watching for file changes with StatReloader
INFO 2024-06-12 21:27:31,015 basehttp 13988 9100 "GET / HTTP/1.1" 200 648
WARNING 2024-06-12 21:27:31,221 log 13988 9100 Not Found: /favicon.ico
WARNING 2024-06-12 21:27:31,222 basehttp 13988 9100 "GET /favicon.ico HTTP/1.1" 404 2760
WARNING 2024-06-12 21:27:45,170 log 13988 15896 Not Found: /f/
WARNING 2024-06-12 21:27:45,171 basehttp 13988 15896 "GET /f/ HTTP/1.1" 404 2807
INFO 2024-06-12 21:27:55,203 basehttp 13988 15896 "GET /f/user/add/ HTTP/1.1" 200 630
INFO 2024-06-12 21:31:15,969 basehttp 13988 15896 "GET /f/user/add/ HTTP/1.1" 200 848
INFO 2024-06-12 21:33:42,661 basehttp 13988 15896 "POST /f/user/add/ HTTP/1.1" 200 892
INFO 2024-06-12 21:33:56,796 basehttp 13988 15896 "POST /f/user/add/ HTTP/1.1" 200 893
INFO 2024-06-12 21:34:59,210 autoreload 13988 15708 D:\Ucheba\django_project\myproject\lec_forms\views.py changed, reloading.
INFO 2024-06-12 21:34:59,746 autoreload 10992 12708 Watching for file changes with StatReloader
INFO 2024-06-12 21:35:05,836 basehttp 10992 1604 "POST /f/user/add/ HTTP/1.1" 200 893
INFO 2024-06-12 21:35:25,306 autoreload 11068 13088 Watching for file changes with StatReloader
INFO 2024-06-12 21:35:30,618 basehttp 11068 744 "POST /f/user/add/ HTTP/1.1" 200 893
INFO 2024-06-12 21:35:32,020 basehttp 11068 744 "POST /f/user/add/ HTTP/1.1" 200 893
INFO 2024-06-12 21:35:37,828 basehttp 11068 744 "POST /f/user/add/ HTTP/1.1" 200 893
INFO 2024-06-12 21:37:48,024 autoreload 11796 15792 Watching for file changes with StatReloader
INFO 2024-06-12 21:37:54,028 basehttp 11796 4752 "GET / HTTP/1.1" 200 648
INFO 2024-06-12 21:37:57,963 basehttp 11796 4752 "POST /f/user/add/ HTTP/1.1" 200 893
INFO 2024-06-12 21:38:20,787 basehttp 11796 4752 "POST /f/user/add/ HTTP/1.1" 200 899
INFO 2024-06-12 22:14:11,693 autoreload 14848 13096 Watching for file changes with StatReloader
INFO 2024-06-12 22:14:22,774 basehttp 14848 14672 "GET /f/forms/ HTTP/1.1" 200 1510
ERROR 2024-06-12 22:15:12,150 log 14848 14672 Internal Server Error: /f/forms/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\lec_forms\views.py", line 28, in many_fields_form
    logger.info({form.cleaned_data})
                ^^^^^^^^^^^^^^^^^^^
TypeError: unhashable type: 'dict'
ERROR 2024-06-12 22:15:12,151 basehttp 14848 14672 "POST /f/forms/ HTTP/1.1" 500 70149
INFO 2024-06-12 22:16:15,202 autoreload 14848 13096 D:\Ucheba\django_project\myproject\lec_forms\views.py changed, reloading.
INFO 2024-06-12 22:16:15,726 autoreload 6864 5920 Watching for file changes with StatReloader
INFO 2024-06-12 22:16:16,934 basehttp 6864 5344 "POST /f/forms/ HTTP/1.1" 200 1583
INFO 2024-06-12 22:16:19,329 basehttp 6864 5344 "POST /f/forms/ HTTP/1.1" 200 1583
INFO 2024-06-12 22:26:54,726 autoreload 6864 5920 D:\Ucheba\django_project\myproject\lec_forms\forms.py changed, reloading.
INFO 2024-06-12 22:26:55,270 autoreload 12668 1460 Watching for file changes with StatReloader
INFO 2024-06-12 22:29:54,076 autoreload 12668 1460 D:\Ucheba\django_project\myproject\lec_forms\views.py changed, reloading.
INFO 2024-06-12 22:29:54,601 autoreload 12832 15472 Watching for file changes with StatReloader
INFO 2024-06-12 22:30:09,635 autoreload 12832 15472 D:\Ucheba\django_project\myproject\lec_forms\views.py changed, reloading.
INFO 2024-06-12 22:30:10,178 autoreload 2092 13748 Watching for file changes with StatReloader
INFO 2024-06-12 22:30:22,012 autoreload 2092 13748 D:\Ucheba\django_project\myproject\lec_forms\views.py changed, reloading.
INFO 2024-06-12 22:30:22,539 autoreload 8432 6340 Watching for file changes with StatReloader
INFO 2024-06-12 22:30:47,395 basehttp 8432 15384 "POST /f/forms/ HTTP/1.1" 200 2293
INFO 2024-06-12 22:31:18,624 basehttp 8432 15384 "POST /f/forms/ HTTP/1.1" 200 2212
INFO 2024-06-12 22:33:20,276 autoreload 2556 10476 Watching for file changes with StatReloader
INFO 2024-06-12 22:33:47,520 basehttp 2556 2772 "POST /f/forms/ HTTP/1.1" 200 2310
INFO 2024-06-12 22:34:45,634 autoreload 2556 10476 D:\Ucheba\django_project\myproject\lec_forms\forms.py changed, reloading.
INFO 2024-06-12 22:34:46,193 autoreload 4508 8788 Watching for file changes with StatReloader
INFO 2024-06-12 22:35:03,387 basehttp 4508 15892 "POST /f/forms/ HTTP/1.1" 200 2310
INFO 2024-06-12 22:40:25,125 autoreload 4508 8788 D:\Ucheba\django_project\myproject\lec_forms\forms.py changed, reloading.
INFO 2024-06-12 22:40:25,632 autoreload 14640 11436 Watching for file changes with StatReloader
INFO 2024-06-12 22:47:07,935 autoreload 14640 11436 D:\Ucheba\django_project\myproject\lec_forms\forms.py changed, reloading.
INFO 2024-06-12 22:47:08,487 autoreload 2916 13640 Watching for file changes with StatReloader
INFO 2024-06-12 23:33:20,656 autoreload 10692 15144 Watching for file changes with StatReloader
INFO 2024-06-12 23:33:22,392 basehttp 10692 16000 "GET / HTTP/1.1" 200 648
INFO 2024-06-12 23:33:28,283 basehttp 10692 16000 "GET /lec/ HTTP/1.1" 200 175
WARNING 2024-06-12 23:33:34,068 log 10692 16000 Not Found: /docs
WARNING 2024-06-12 23:33:34,069 basehttp 10692 16000 "GET /docs HTTP/1.1" 404 2739
INFO 2024-06-12 23:33:52,299 basehttp 10692 16000 "GET /f/user/ HTTP/1.1" 200 887
INFO 2024-06-12 23:34:12,653 basehttp 10692 16000 "POST /f/user/ HTTP/1.1" 200 1030
INFO 2024-06-12 23:34:28,837 basehttp 10692 16000 "POST /f/user/ HTTP/1.1" 200 946
INFO 2024-06-12 23:47:10,554 autoreload 12176 13952 Watching for file changes with StatReloader
INFO 2024-06-12 23:48:58,489 basehttp 12176 13444 "GET /f/upload/ HTTP/1.1" 200 654
ERROR 2024-06-12 23:49:06,440 log 12176 13444 Internal Server Error: /f/upload/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\lec_forms\views.py", line 57, in upload_image
    if form.is_valid():
       ^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\forms\forms.py", line 197, in is_valid
    return self.is_bound and not self.errors
                                 ^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\forms\forms.py", line 192, in errors
    self.full_clean()
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\forms\forms.py", line 327, in full_clean
    self._clean_fields()
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\forms\forms.py", line 337, in _clean_fields
    value = field.clean(value, bf.initial)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\forms\fields.py", line 688, in clean
    return super().clean(data)
           ^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\forms\fields.py", line 203, in clean
    value = self.to_python(value)
            ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\forms\fields.py", line 715, in to_python
    from PIL import Image
ModuleNotFoundError: No module named 'PIL'
ERROR 2024-06-12 23:49:06,442 basehttp 12176 13444 "POST /f/upload/ HTTP/1.1" 500 98152
INFO 2024-06-12 23:51:23,381 autoreload 8764 13260 Watching for file changes with StatReloader
ERROR 2024-06-12 23:51:32,136 log 8764 14712 Internal Server Error: /f/upload/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\lec_forms\views.py", line 58, in upload_image
    image = form.cleaned_data['image']
            ~~~~~~~~~~~~~~~~~^^^^^^^^^
KeyError: 'image'
ERROR 2024-06-12 23:51:32,137 basehttp 8764 14712 "POST /f/upload/ HTTP/1.1" 500 69895
INFO 2024-06-12 23:56:52,705 basehttp 8764 10944 "GET /f/upload/ HTTP/1.1" 200 654
INFO 2024-06-12 23:56:53,076 autoreload 8764 13260 D:\Ucheba\django_project\myproject\lec_forms\forms.py changed, reloading.
INFO 2024-06-12 23:56:53,632 autoreload 16324 15696 Watching for file changes with StatReloader
INFO 2024-06-12 23:56:55,488 basehttp 16324 1856 "GET /f/upload/ HTTP/1.1" 200 658
INFO 2024-06-12 23:57:00,881 basehttp 16324 1856 "POST /f/upload/ HTTP/1.1" 200 658
INFO 2024-06-13 19:22:00,583 autoreload 15408 2820 Watching for file changes with StatReloader
INFO 2024-06-13 19:22:02,708 basehttp 15408 15784 "GET / HTTP/1.1" 200 648
WARNING 2024-06-13 19:22:03,030 log 15408 15784 Not Found: /favicon.ico
WARNING 2024-06-13 19:22:03,030 basehttp 15408 15784 "GET /favicon.ico HTTP/1.1" 404 2760
INFO 2024-06-13 19:23:05,105 autoreload 15408 2820 D:\Ucheba\django_project\myproject\myproject\urls.py changed, reloading.
INFO 2024-06-13 19:23:05,649 autoreload 10692 3688 Watching for file changes with StatReloader
INFO 2024-06-13 19:23:07,788 basehttp 10692 6664 "GET / HTTP/1.1" 200 648
WARNING 2024-06-13 19:23:14,141 log 10692 6664 Not Found: /fs
WARNING 2024-06-13 19:23:14,141 basehttp 10692 6664 "GET /fs HTTP/1.1" 404 2836
WARNING 2024-06-13 19:23:28,130 log 10692 6664 Not Found: /fs/coin_flip/
WARNING 2024-06-13 19:23:28,131 basehttp 10692 6664 "GET /fs/coin_flip/ HTTP/1.1" 404 3293
WARNING 2024-06-13 19:23:36,682 log 10692 6664 Not Found: /fs/coin_flip/5
WARNING 2024-06-13 19:23:36,682 basehttp 10692 6664 "GET /fs/coin_flip/5 HTTP/1.1" 404 3296
INFO 2024-06-13 19:24:13,340 autoreload 10692 3688 D:\Ucheba\django_project\myproject\form_seminar\urls.py changed, reloading.
INFO 2024-06-13 19:24:13,872 autoreload 14796 1012 Watching for file changes with StatReloader
INFO 2024-06-13 19:27:25,684 autoreload 4340 13828 Watching for file changes with StatReloader
INFO 2024-06-13 19:27:27,029 basehttp 4340 2396 "GET / HTTP/1.1" 200 648
WARNING 2024-06-13 19:27:36,685 log 4340 2396 Not Found: /fs/flip/
WARNING 2024-06-13 19:27:36,685 basehttp 4340 2396 "GET /fs/flip/ HTTP/1.1" 404 3329
ERROR 2024-06-13 19:27:39,595 log 4340 2396 Internal Server Error: /fs/flip/5
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\form_seminar\views.py", line 9, in flip_coin
    return render(request, "form_seminar/games_f.html", context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: form_seminar/games_f.html
ERROR 2024-06-13 19:27:39,599 basehttp 4340 2396 "GET /fs/flip/5 HTTP/1.1" 500 86836
INFO 2024-06-13 19:28:03,767 basehttp 4340 2396 "GET /fs/flip/5 HTTP/1.1" 200 416
INFO 2024-06-13 19:28:21,422 basehttp 4340 2396 "GET /fs/flip_t/5 HTTP/1.1" 200 377
INFO 2024-06-13 19:28:28,075 basehttp 4340 2396 "GET /fs/flip_r/5 HTTP/1.1" 200 381
INFO 2024-06-13 19:31:23,718 basehttp 4340 2396 "GET /fs/flip_t/5 HTTP/1.1" 200 377
INFO 2024-06-13 19:31:30,667 basehttp 4340 2396 "GET /fs/flip/5 HTTP/1.1" 200 414
INFO 2024-06-13 19:32:32,376 autoreload 4340 13828 D:\Ucheba\django_project\myproject\form_seminar\views.py changed, reloading.
INFO 2024-06-13 19:32:32,965 autoreload 12764 13696 Watching for file changes with StatReloader
INFO 2024-06-13 19:40:30,404 autoreload 12764 13696 D:\Ucheba\django_project\myproject\form_seminar\views.py changed, reloading.
INFO 2024-06-13 19:40:30,953 autoreload 15220 13480 Watching for file changes with StatReloader
INFO 2024-06-13 19:41:40,371 basehttp 15220 12476 "GET / HTTP/1.1" 200 648
WARNING 2024-06-13 19:41:52,643 log 15220 12476 Not Found: /fs/flip_t/
WARNING 2024-06-13 19:41:52,643 basehttp 15220 12476 "GET /fs/flip_t/ HTTP/1.1" 404 3335
INFO 2024-06-13 19:41:57,367 basehttp 15220 12476 "GET /fs/flip_t/5 HTTP/1.1" 200 437
INFO 2024-06-13 19:42:00,507 basehttp 15220 12476 "POST /fs/flip_t/5 HTTP/1.1" 200 437
INFO 2024-06-13 19:43:18,860 autoreload 15220 13480 D:\Ucheba\django_project\myproject\form_seminar\urls.py changed, reloading.
INFO 2024-06-13 19:43:19,427 autoreload 12792 10292 Watching for file changes with StatReloader
INFO 2024-06-13 19:43:27,011 basehttp 12792 15340 "GET /fs/flip_f/ HTTP/1.1" 200 936
INFO 2024-06-13 19:43:42,273 basehttp 12792 15340 "POST /fs/flip_f/ HTTP/1.1" 200 437
INFO 2024-06-13 19:46:50,002 autoreload 12792 10292 D:\Ucheba\django_project\myproject\form_seminar\views.py changed, reloading.
INFO 2024-06-13 19:46:50,549 autoreload 15836 11656 Watching for file changes with StatReloader
INFO 2024-06-13 19:46:53,071 basehttp 15836 15816 "GET /fs/flip_f/ HTTP/1.1" 200 936
INFO 2024-06-13 19:46:59,173 basehttp 15836 15816 "POST /fs/flip_f/ HTTP/1.1" 200 407
INFO 2024-06-13 19:47:05,953 basehttp 15836 15816 "GET /fs/flip_f/ HTTP/1.1" 200 936
INFO 2024-06-13 19:47:10,369 basehttp 15836 15816 "POST /fs/flip_f/ HTTP/1.1" 200 512
INFO 2024-06-13 19:47:17,571 basehttp 15836 15816 "GET /fs/flip_f/ HTTP/1.1" 200 936
INFO 2024-06-13 19:47:21,209 basehttp 15836 15816 "POST /fs/flip_f/ HTTP/1.1" 200 386
INFO 2024-06-13 19:47:25,523 basehttp 15836 15816 "GET /fs/flip_f/ HTTP/1.1" 200 936
INFO 2024-06-15 12:49:53,616 autoreload 6436 2588 Watching for file changes with StatReloader
INFO 2024-06-15 12:49:56,593 basehttp 6436 2096 "GET / HTTP/1.1" 200 648
INFO 2024-06-15 12:50:03,553 basehttp 6436 2096 "GET /hw/pre/ HTTP/1.1" 200 1044
INFO 2024-06-15 12:50:05,587 basehttp 6436 2096 "GET /hw/orders/ HTTP/1.1" 200 369
INFO 2024-06-15 12:50:08,240 basehttp 6436 2096 "GET /hw/orders/ HTTP/1.1" 200 369
INFO 2024-06-15 13:48:09,411 autoreload 8704 13972 Watching for file changes with StatReloader
INFO 2024-06-15 13:48:10,522 basehttp 8704 4540 "GET / HTTP/1.1" 200 648
INFO 2024-06-15 13:48:18,581 basehttp 8704 4540 "GET /hw/pre/ HTTP/1.1" 200 1044
ERROR 2024-06-15 13:48:20,901 log 8704 4540 Internal Server Error: /hw/orders/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 45, in orders
    return render(request, "hw_logik_and_template/orders.html", context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 325, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 241, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 478, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\urls\resolvers.py", line 848, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'orders' with arguments '('',)' not found. 1 pattern(s) tried: ['hw/orders/(?P<client_id>[0-9]+)\\Z']
ERROR 2024-06-15 13:48:20,906 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 500 178388
INFO 2024-06-15 13:49:05,097 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 566
INFO 2024-06-15 13:50:10,528 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:50:19,763 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:50:21,387 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:50:57,934 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:50:58,733 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:50:59,389 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:51:00,223 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:51:00,802 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:51:01,335 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:51:01,885 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:51:38,760 basehttp 8704 4540 "GET /hw/orders/ HTTP/1.1" 200 590
INFO 2024-06-15 13:51:39,298 autoreload 8704 13972 D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py changed, reloading.
INFO 2024-06-15 13:51:39,837 autoreload 5936 11812 Watching for file changes with StatReloader
INFO 2024-06-15 13:51:39,931 basehttp 5936 15244 "GET /hw/orders/ HTTP/1.1" 200 609
INFO 2024-06-15 14:30:37,441 autoreload 15352 4692 Watching for file changes with StatReloader
INFO 2024-06-15 14:30:38,729 basehttp 15352 12696 "GET / HTTP/1.1" 200 648
INFO 2024-06-15 14:31:33,426 basehttp 15352 12696 "GET / HTTP/1.1" 200 648
INFO 2024-06-15 14:31:39,460 basehttp 15352 12696 "GET /hw/pre/ HTTP/1.1" 200 1044
INFO 2024-06-15 14:31:41,749 basehttp 15352 12696 "GET /hw/orders/ HTTP/1.1" 200 1313
ERROR 2024-06-15 14:31:45,923 log 15352 12696 Internal Server Error: /hw/orders/1
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 25, in orders
    order_week = order.filter(order_date__gte=today - timedelta(days=7))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1600, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1632, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1482, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1294, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1759, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'order_date' into field. Choices are: client, client_id, date_of_order, id, price, products
ERROR 2024-06-15 14:31:45,926 basehttp 15352 12696 "GET /hw/orders/1 HTTP/1.1" 500 111291
ERROR 2024-06-15 14:31:59,814 log 15352 12696 Internal Server Error: /hw/orders/4
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 25, in orders
    order_week = order.filter(order_date__gte=today - timedelta(days=7))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1600, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1632, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1482, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1294, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1759, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'order_date' into field. Choices are: client, client_id, date_of_order, id, price, products
ERROR 2024-06-15 14:31:59,816 basehttp 15352 12696 "GET /hw/orders/4 HTTP/1.1" 500 110843
INFO 2024-06-16 11:07:45,065 autoreload 8016 13464 Watching for file changes with StatReloader
INFO 2024-06-16 11:07:49,755 basehttp 8016 10000 "GET / HTTP/1.1" 200 648
WARNING 2024-06-16 11:07:49,988 log 8016 10000 Not Found: /favicon.ico
WARNING 2024-06-16 11:07:49,989 basehttp 8016 10000 "GET /favicon.ico HTTP/1.1" 404 2966
WARNING 2024-06-16 11:07:49,994 log 8016 10196 Not Found: /favicon.ico
WARNING 2024-06-16 11:07:49,995 basehttp 8016 10196 "GET /favicon.ico HTTP/1.1" 404 2966
INFO 2024-06-16 11:07:58,306 basehttp 8016 5596 "GET /hw/pre/ HTTP/1.1" 200 1044
INFO 2024-06-16 11:08:00,560 basehttp 8016 5596 "GET /hw/orders/ HTTP/1.1" 200 1313
INFO 2024-06-16 11:08:35,209 basehttp 8016 5596 "GET /hw/orders/ HTTP/1.1" 200 1233
INFO 2024-06-16 11:08:38,332 basehttp 8016 5596 "GET /hw/orders/ HTTP/1.1" 200 1233
ERROR 2024-06-16 11:08:42,813 log 8016 5596 Internal Server Error: /hw/orders/1
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 25, in orders
    order_week = order.filter(order_date__gte=today - timedelta(days=7))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1600, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1632, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1482, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1294, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1759, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'order_date' into field. Choices are: client, client_id, date_of_order, id, price, products
ERROR 2024-06-16 11:08:42,817 basehttp 8016 5596 "GET /hw/orders/1 HTTP/1.1" 500 111273
ERROR 2024-06-16 11:09:55,917 log 8016 4648 Internal Server Error: /hw/orders/3
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 25, in orders
    order_week = order.filter(order_date__gte=today - timedelta(days=7))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1600, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1632, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1482, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1294, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1759, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'order_date' into field. Choices are: client, client_id, date_of_order, id, price, products
ERROR 2024-06-16 11:09:55,919 basehttp 8016 4648 "GET /hw/orders/3 HTTP/1.1" 500 110792
INFO 2024-06-16 11:10:01,549 basehttp 8016 4648 "GET /hw/orders/ HTTP/1.1" 200 1313
ERROR 2024-06-16 11:10:02,738 log 8016 4648 Internal Server Error: /hw/orders/3
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 25, in orders
    order_week = order.filter(order_date__gte=today - timedelta(days=7))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1600, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1632, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1482, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1294, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1759, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'order_date' into field. Choices are: client, client_id, date_of_order, id, price, products
ERROR 2024-06-16 11:10:02,740 basehttp 8016 4648 "GET /hw/orders/3 HTTP/1.1" 500 110792
ERROR 2024-06-16 11:10:10,243 log 8016 4648 Internal Server Error: /hw/orders/3
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 25, in orders
    order_week = order.filter(order_date__gte=today - timedelta(days=7))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1600, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1632, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1482, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1294, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1759, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'order_date' into field. Choices are: client, client_id, date_of_order, id, price, products
ERROR 2024-06-16 11:10:10,245 basehttp 8016 4648 "GET /hw/orders/3 HTTP/1.1" 500 110801
ERROR 2024-06-16 11:11:06,327 log 8016 4648 Internal Server Error: /hw/orders/3
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 25, in orders
    order_week = order.filter(order_date__gte=today - timedelta(days=7))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1600, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1632, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1482, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1294, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1759, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'order_date' into field. Choices are: client, client_id, date_of_order, id, price, products
ERROR 2024-06-16 11:11:06,329 basehttp 8016 4648 "GET /hw/orders/3 HTTP/1.1" 500 110792
ERROR 2024-06-16 11:11:36,281 log 8016 4648 Internal Server Error: /hw/orders/3
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 25, in orders
    order_week = order.filter(date_of_order__gte=today - timedelta(days=7))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1600, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1632, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1482, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1294, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\db\models\sql\query.py", line 1759, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'order_date' into field. Choices are: client, client_id, date_of_order, id, price, products
ERROR 2024-06-16 11:11:36,283 basehttp 8016 4648 "GET /hw/orders/3 HTTP/1.1" 500 110950
INFO 2024-06-16 11:11:37,030 autoreload 8016 13464 D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py changed, reloading.
INFO 2024-06-16 11:11:37,577 autoreload 5856 7384 Watching for file changes with StatReloader
INFO 2024-06-16 11:11:37,921 basehttp 5856 4260 "GET /hw/orders/3 HTTP/1.1" 200 970
INFO 2024-06-16 11:11:53,621 basehttp 5856 4260 "GET /hw/orders/1 HTTP/1.1" 200 1382
INFO 2024-06-16 11:14:17,907 basehttp 5856 4260 "GET /hw/orders/3 HTTP/1.1" 200 970
INFO 2024-06-16 11:14:22,784 basehttp 5856 4260 "GET /hw/orders/3 HTTP/1.1" 200 970
INFO 2024-06-16 11:14:27,661 basehttp 5856 4260 "GET /hw/orders/1 HTTP/1.1" 200 1382
WARNING 2024-06-16 11:18:09,634 log 5856 4260 Not Found: /hw/
WARNING 2024-06-16 11:18:09,634 basehttp 5856 4260 "GET /hw/ HTTP/1.1" 404 3379
INFO 2024-06-16 11:18:20,230 basehttp 5856 4260 "GET /hw/pre/ HTTP/1.1" 200 1044
INFO 2024-06-16 11:18:21,843 basehttp 5856 4260 "GET /hw/orders/ HTTP/1.1" 200 1309
INFO 2024-06-16 11:18:24,054 basehttp 5856 4260 "GET /hw/orders/1 HTTP/1.1" 200 1475
INFO 2024-06-16 11:18:34,730 basehttp 5856 4260 "GET /hw/orders/4 HTTP/1.1" 200 1058
INFO 2024-06-16 11:18:44,478 basehttp 5856 4260 "GET /hw/orders/3 HTTP/1.1" 200 1019
INFO 2024-06-16 11:21:50,351 autoreload 5856 7384 D:\Ucheba\django_project\myproject\hw_logik_and_template\models.py changed, reloading.
INFO 2024-06-16 11:21:50,875 autoreload 10564 13448 Watching for file changes with StatReloader
INFO 2024-06-16 11:22:04,365 basehttp 10564 11812 "GET /hw/orders/1 HTTP/1.1" 200 1475
INFO 2024-06-16 11:23:03,642 autoreload 10564 13448 D:\Ucheba\django_project\myproject\hw_logik_and_template\models.py changed, reloading.
INFO 2024-06-16 11:23:04,166 autoreload 2824 8 Watching for file changes with StatReloader
INFO 2024-06-16 11:27:17,595 basehttp 2824 5288 "GET /hw/orders/ HTTP/1.1" 200 1651
INFO 2024-06-16 11:27:18,678 basehttp 2824 5288 "GET /hw/orders/ HTTP/1.1" 200 1651
INFO 2024-06-16 11:27:31,008 basehttp 2824 5288 "GET /hw/orders/1 HTTP/1.1" 200 2158
INFO 2024-06-16 11:30:09,618 autoreload 2824 8 D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py changed, reloading.
INFO 2024-06-16 11:30:10,115 autoreload 8960 2572 Watching for file changes with StatReloader
INFO 2024-06-16 11:30:10,417 basehttp 8960 12064 "GET /hw/orders/1 HTTP/1.1" 200 2158
INFO 2024-06-16 11:30:39,414 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1308
INFO 2024-06-16 11:30:42,777 basehttp 8960 12064 "GET /hw/orders/4 HTTP/1.1" 200 1057
INFO 2024-06-16 11:32:11,116 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1440
INFO 2024-06-16 11:32:19,796 basehttp 8960 12064 "GET /hw/orders/3 HTTP/1.1" 200 1018
ERROR 2024-06-16 11:35:32,308 log 8960 12064 Internal Server Error: /hw/orders/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 179, in translate_token
    op = OPERATORS[token]
         ~~~~~~~~~^^^^^^^
KeyError: '-'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 45, in orders
    return render(request, "hw_logik_and_template/orders.html", context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 960, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 860, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 959, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 892, in __init__
    super().__init__(*args, **kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 170, in __init__
    mapped_tokens.append(self.translate_token(token))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 181, in translate_token
    return self.create_var(token)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 895, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '-' from '-'
ERROR 2024-06-16 11:35:32,318 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 500 277587
ERROR 2024-06-16 11:35:35,812 log 8960 12064 Internal Server Error: /hw/orders/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 179, in translate_token
    op = OPERATORS[token]
         ~~~~~~~~~^^^^^^^
KeyError: '-'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 45, in orders
    return render(request, "hw_logik_and_template/orders.html", context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 960, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 860, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 959, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 892, in __init__
    super().__init__(*args, **kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 170, in __init__
    mapped_tokens.append(self.translate_token(token))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 181, in translate_token
    return self.create_var(token)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 895, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '-' from '-'
ERROR 2024-06-16 11:35:35,817 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 500 277450
ERROR 2024-06-16 11:36:08,711 log 8960 12064 Internal Server Error: /hw/orders/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 179, in translate_token
    op = OPERATORS[token]
         ~~~~~~~~~^^^^^^^
KeyError: '-'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 45, in orders
    return render(request, "hw_logik_and_template/orders.html", context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 960, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 860, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 959, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 892, in __init__
    super().__init__(*args, **kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 170, in __init__
    mapped_tokens.append(self.translate_token(token))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 181, in translate_token
    return self.create_var(token)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 895, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '-' from '-'
ERROR 2024-06-16 11:36:08,718 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 500 278851
ERROR 2024-06-16 11:36:10,462 log 8960 12064 Internal Server Error: /hw/orders/
Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 179, in translate_token
    op = OPERATORS[token]
         ~~~~~~~~~^^^^^^^
KeyError: '-'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\django_project\myproject\hw_logik_and_template\views.py", line 45, in orders
    return render(request, "hw_logik_and_template/orders.html", context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 960, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 860, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 959, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 892, in __init__
    super().__init__(*args, **kwargs)
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 170, in __init__
    mapped_tokens.append(self.translate_token(token))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\smartif.py", line 181, in translate_token
    return self.create_var(token)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\defaulttags.py", line 895, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ucheba\venvs\venv_gb_django\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '-' from '-'
ERROR 2024-06-16 11:36:10,467 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 500 278851
INFO 2024-06-16 11:38:20,818 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1436
INFO 2024-06-16 11:38:22,040 basehttp 8960 12064 "GET /hw/orders/1 HTTP/1.1" 200 1504
INFO 2024-06-16 11:38:28,015 basehttp 8960 12064 "GET /hw/orders/1 HTTP/1.1" 200 1504
INFO 2024-06-16 11:40:34,027 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1436
INFO 2024-06-16 11:40:35,814 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1436
INFO 2024-06-16 11:40:36,575 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1436
INFO 2024-06-16 11:40:37,183 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1436
INFO 2024-06-16 11:40:37,693 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1436
INFO 2024-06-16 11:40:39,657 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1436
INFO 2024-06-16 11:40:40,430 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1436
INFO 2024-06-16 11:41:23,211 basehttp 8960 12064 "GET /hw/orders/4 HTTP/1.1" 200 1093
INFO 2024-06-16 11:42:01,194 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1520
INFO 2024-06-16 11:42:07,157 basehttp 8960 12064 "GET /hw/orders/1 HTTP/1.1" 200 1546
INFO 2024-06-16 11:43:54,247 basehttp 8960 12064 "GET /hw/orders/1 HTTP/1.1" 200 1242
INFO 2024-06-16 11:44:34,778 basehttp 8960 12064 "GET /hw/pre/ HTTP/1.1" 200 1044
INFO 2024-06-16 11:45:01,410 basehttp 8960 12064 "GET /hw/orders/ HTTP/1.1" 200 1322
INFO 2024-06-16 11:48:46,348 autoreload 2636 9180 Watching for file changes with StatReloader
INFO 2024-06-16 11:48:47,336 basehttp 2636 3584 "GET / HTTP/1.1" 200 648
INFO 2024-06-16 11:48:51,237 basehttp 2636 3584 "GET /hw/pre/ HTTP/1.1" 200 1044
INFO 2024-06-16 11:48:53,410 basehttp 2636 3584 "GET /hw/orders/ HTTP/1.1" 200 3515
INFO 2024-06-16 11:49:00,475 basehttp 2636 3584 "GET /hw/orders/2 HTTP/1.1" 200 1460
INFO 2024-06-16 11:49:39,657 basehttp 2636 3584 "GET /hw/orders/1 HTTP/1.1" 200 2235
INFO 2024-06-16 11:49:54,429 basehttp 2636 3584 "GET /hw/orders/4 HTTP/1.1" 200 928
INFO 2024-06-16 11:50:02,019 basehttp 2636 3584 "GET /hw/orders/1 HTTP/1.1" 200 2235
INFO 2024-06-16 11:50:07,377 basehttp 2636 3584 "GET /hw/orders/3 HTTP/1.1" 200 1843
INFO 2024-06-16 11:50:16,320 basehttp 2636 3584 "GET /hw/orders/3 HTTP/1.1" 200 1843
INFO 2024-06-16 11:55:41,581 basehttp 2636 2288 "GET /hw/orders/ HTTP/1.1" 200 3307
INFO 2024-06-16 11:55:42,474 basehttp 2636 2288 "GET /hw/orders/ HTTP/1.1" 200 3307
